packages/react-scripts/fixtures/kitchensink/src/features/webpack/CssModulesInclusion.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+import React from 'react';
+import styles from './assets/style.module.css';
+
+export default () => (
+  <p className={styles.cssModuleInclusion}>We love useless text.</p>
```

Comment:
We don't have an integration test demonstrating that the class name actually gets applied. Can you please add one in the `integration` folder?